### PR TITLE
turn on some ability_spec tests

### PR DESCRIPTION
**Pull request by pjfanning**
I've turned on some of the assertions and will try to get the other assertions fixed up soon. I just want to see if I'm in the right ball-park with making rspec changes.

### CODE REVIEW

**Comment by dottorer** [comment]
Nice start! @phil-monroe can take a look and help you get started in rspec...

**Comment by phil-monroe** [comment]
@pjfanning definitely a good start. Let me know when you are ready for a review!

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -10,15 +10,15 @@
 
     context "as a guest" do
       let(:user){ nil }
-      it{ should     be_able_to(:read, circle) }
+      xit{ should     be_able_to(:read, circle) }
```
Comment (pjfanning) [inline spec/models/ability_spec.rb]: In ability.rb, we have
`    can :read,   Circle do |circle|
      circle.users.include? user
    end`
So it appears tom me that guests cannot read circles. If this is the case, should we change ability.rb to allow all users to read circles?

Comment (dottorer) [inline spec/models/ability_spec.rb]: Currently I am not sure what a 'guest' would signify. I think soon we will have new users that have not been approved by the circle administrator and those would have limited view right to the circle info. They should be able to see basic circle data but nothing more (i.e. sign up for tasks or anything like that).

**Comment by pjfanning** [comment]
Apologies, I misread the text on the merge button on this review and thought it was to add more content from my branch to the pull request. The changes are just in test code but feel free to revert them if they are problematic.

**Comment by pjfanning** [comment]
before(:each) { task.volunteers << user; task.save }
This is leading to exceptions when the ability_spec.rb.
The SQL seems a little off to my eye - there appear to be 7 items in the row data but only 4 columns in the insert statement.

     Failure/Error: before(:each) { task.volunteers << user; task.save }
     ActiveRecord::StatementInvalid:
       PG::NotNullViolation: ERROR:  null value in column "role_type" violates not-null constraint
       DETAIL:  Failing row contains (4, null, 32, 15, 2016-02-08 21:49:20.742735, 2016-02-08 21:49:20.742735).
       : INSERT INTO "task_roles" ("task_id", "user_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"

**Pull request merged**